java-springboot/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,6 @@
+p44:
+  application:
+    name: ${spring.application.name}
```

Comment:
I am not seeing where the reference to the application name is being defined. 
I might be blind or maybe It's that part of the test to provide?

Reply:
A mistake I will correct. I need to set this.